code/src/components/Thought-Item.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Emoji from '../utils/Emoji'
+import moment from 'moment' 
+
+
+const ThoughtItem = ({ thought, onLikesIncrease }) => {
+    return (
+        <main>
+            <div className="thought-container">
+                <p className="thought-text">{thought.message}</p>
+              <div className="likes-container">
+                    <div className="likes-wrapper">
+                    <button className="likebtn" type="button" onClick={() => onLikesIncrease(thought._id)}>
+                    {' '}
```

Comment:
What does this and the one on line 17 do? I haven't used this myself so that's why I'm curious :)